Automattic/jetpack
# Carousel: hide carousel controls onZoom

Fixes https://github.com/Automattic/view-design/issues/289

#### Changes proposed in this Pull Request

On resize of an image in the carousel we're hiding the bottom bar and the close button. 

We'll show them again when the image's zoom is reset or the image is swiped/tapped.

https://user-images.githubusercontent.com/6458278/123911928-beeb3a80-d9bf-11eb-9d67-83044b534c3b.MP4

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

1. In your test site enable the Carousel module: `/wp-admin/admin.php?page=jetpack#/writing`
2. Add an image gallery to a post and publish
3. Zoom in on an image. The footer and close button should be hidden.
4. Clicking on the container, reseting the zoom or changing slides should show the footer and close button again.
5. Try in both desktop and mobile (also pinch and zoom/double and single taps)
